app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -139,6 +139,13 @@
     // Parsed RADIUS server addresses
     protected InetAddress radiusIpAddress;
 
+    // To check response from RADIUS SERVER
+    boolean waitRadiusResponse;
```

Comment:
How can there be just 1 boolean flag for all the ongoing sessions?

Reply:
Completed

Reply:
Mapped this to stateMachine